app/src/Components/ListTable/ListTableRowButtons.js

```diff
@@ -29,23 +32,34 @@ export const ListTableRowButtons = ({buttons, value}) => {
 }
 
 function _createButton(button, value) {
-  switch(button.function) {
+  switch(button.name) {
     case 'edit':
       return (
           <IconButton key={'edit_' + value} component={Link} to={button.parameter + '/edit/' + value} aria-label="Delete" color="primary">
             <EditIcon/>
           </IconButton>
       );
-    default:
+    case 'delete':
       return(
-        <IconButton key={'delete_' + value} onClick={() => _delete(button.parameter, value)} aria-label="Delete" color="primary">
-          <DeleteIcon/>
-        </IconButton>
+        <Mutation
+          key={'delete_' + value}
+          mutation={button.parameter.delete()}
+          variables={{id: value}}
+          refetchQueries={[{query: button.parameter.getListPage()}]}
+        >
+        { (deleteFunc, { loading, error }) => {
+         if (loading) return (<p>Loading...</p>);
+         if (error) return (<p>Error...</p>);
+         return(
+           <IconButton key={'delete_' + value} onClick={() => {deleteFunc()}} aria-label="Delete" color="primary">
+            <DeleteIcon/>
```

Comment:
Change `<DeleteIcon/>` to `<DeleteIcon />`

Reply:
I ususally dont put space before />.
This is more of a lint issue and, if we are to update it, has to be updated project-wide